UDV-Core/src/Utils/BaseDemo/js/BaseDemo.js

```diff
@@ -482,7 +482,7 @@ export class BaseDemo {
         itowns.CameraUtils.transformCameraToLookAtTarget(this.view, this.view.camera.camera3D, p);
 
         // Controls
-        this.controls = new itowns.PlanarControls(this.view, {});
+        this.controls = new itowns.PlanarControls(this.view, {maxZenithAngle: 180, groundLevel: -100, handleCollision: false});
```

Comment:
Maybe this should be moved to `DocumentCreationWindow._enterEditMode()` ?

Reply:
I don't think so, because the positioning and the visualization of the documents should have the same parameters (at least of these three options). Otherwise, we could have situations where a document is placed somewhere the camera can't reach with the default settings (too close to the ground, zenith angle out of bounds, etc.), and in this case the visualization is broken.

Reply:
Yep you're right, my bad.